app/src/main/java/com/islamversity/reyan/service/MyFirebaseMessagingService.kt

```diff
@@ -0,0 +1,101 @@
+package com.islamversity.reyan.service
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.media.RingtoneManager
+import android.os.Build
+import android.os.Bundle
+import androidx.core.app.NotificationCompat
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import com.islamversity.reyan.MainActivity
+import com.islamversity.reyan.R
+import com.islamversity.reyan.service.NotificationDataType.Companion.NOTIFICATION_DATA_KEY
+import timber.log.Timber
+
+
+class MyFirebaseMessagingService: FirebaseMessagingService() {
+
+
+    override fun onNewToken(token: String) {
+        Timber.d("Notification : Refreshed token: $token")
+        sendFCMTokenToServer(token)
+    }
+
+    private fun sendFCMTokenToServer(token: String) {
+        // Cache.set(CacheKeys.Firebase.FCM_TOKEN_FIREBASE, token)
+        Timber.d("Notification : sendRegistrationTokenToServer($token)")
+    }
+
+    override fun onMessageReceived(remoteMessage: RemoteMessage) {
+        Timber.d("Notification : From: ${remoteMessage.from}")
```

Comment:
we don't use `Timber`.
the dependency is left here mistakenly
you can use `Logger`

Reply:
why we do not use Timber ?

Reply:
because of KMP